### ISSUE TITLE
hoistPatternLet ignoring patterns with _

### ISSUE THREAD

**Issue by proxi** [nicklockwood/SwiftFormat#479]
Version 0.40.11

`hoistPatternLet` does not transforms patterns with _:

```
enum Foo {
    case foo(Int)
    case bar(Int, Int)
}

switch self {
    case .foo(let x): // -> case let .foo(x):
        break

    case .bar(let x, let y): // -> case let .bar(x, y):
        break

    case .bar(let x, _): // Does not change.
        break
}

```

**Comment by nicklockwood** [comment]
@proxi this is now fixed on develop. The fix will be included in the next release.

**Comment by proxi** [comment]
👏 Thank you, Nick.

**Comment by nicklockwood** [comment]
@proxi this was fixed in 0.40.13.
